collections/src/commonTest/kotlin/AtomicMapTests.kt

```diff
@@ -0,0 +1,25 @@
+package com.juul.tuulbox.collections
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.joinAll
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+public class AtomicMapTests {
+
+    @Test
+    public fun atomicMap_concurrentMutateBlocks_doesNotLoseWrites() = runTest {
```

Comment:
I think there're actually two tests here, or--at least--you're utilizing more than one unit of functionality here in this test. I think this test is actually testing `mutate`, but `atomicMapOf` is also being used. I get that it's a convenience function for initialization, but it does muddy the testing waters.

Reply:
IMHO, `atomicMapOf` is just syntax sugar (doesn't have any logic branches of its own) so is fine to utilize in a test alongside other testing operations.

Could have another test to explicitly test `atomicMapOf`, but I don't that that is necessary. 🤷